test/fail_compilation/fail15896.d

```diff
@@ -0,0 +1,18 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -de
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail15896.d(11): Deprecation: module `imports.imp15896` member `thebar` is not accessible
```

Comment:
s/accessible/visible/
Accessibility is the old protection scheme that will soon be removed.
Also check the other visibility messages, they should be more sth. like `X of mod M is not visible from mod N` or so.